infisical/provider/provider.go

```diff
@@ -80,9 +94,15 @@ func (p *infisicalProvider) Configure(ctx context.Context, req provider.Configur
 		resp.Diagnostics.AddError("No authentication credentials provided", "You must define service_token field of the provider")
 	}
 
-	serviceToken := os.Getenv("INFISICAL_SERVICE_TOKEN")
 	host := os.Getenv("INFISICAL_HOST")
 
+	// Service Token
+	serviceToken := os.Getenv("INFISICAL_SERVICE_TOKEN")
+
+	// Machine Identity
+	clientId := os.Getenv("INFISICAL_CLIENT_ID")
```

Comment:
This should be specifc to the auth name. We should include UNIVERSAL_AUTH in the name. Something like `INFISICAL_UNIVERSAL_AUTH_CLIENT_ID` would be good. Same for client secret